lib/openshift-rest-client.js

```diff
@@ -85,7 +85,7 @@ async function openshiftClient (settings = {}) {
         const user = config.auth.username || config.auth.user;
         const password = config.auth.password || config.auth.pass;
 
-        const accessToken = await getTokenFromBasicAuth({ insecureSkipTlsVerify, url, user, password, authUrl });
+        const accessToken = config.auth.token ? config.auth.token : await getTokenFromBasicAuth({ insecureSkipTlsVerify, url, user, password, authUrl });
```

Comment:
the only way to get to here is if you are also passing in a 'user' or 'username' in the `config.auth`,  so if your config object looked like this:
```
{
  auth: {
    token: 'asawefwfwf'
  }
}
```
then this code would't get run.  probably need to add a check a few lines up that a token is being passed